pkg/utils/nodeclass/nodeclass.go

```diff
@@ -158,6 +159,19 @@ func NewBlockDeviceMappings(bdms []*v1alpha1.BlockDeviceMapping) []*v1beta1.Bloc
 	})
 }
 
+func NewInstanceStoreConfiguration(option *v1alpha1.InstanceStoreConfigurationOption) *v1beta1.InstanceStoreConfigurationOption {
```

Comment:
Hopefully shouldn't need this now that we are dropping alpha support 🤞

Reply:
👍  rebased / dropped alpha files in https://github.com/aws/karpenter/commit/5cc1de801310e130bf6332da9ac71d94f4dd9228